flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Utils/arinc429.js

```diff
@@ -0,0 +1,39 @@
+class Arinc429Word {
+    constructor(value) {
+        Arinc429Word.f64View[0] = value;
+
+        this._ssm = Arinc429Word.u32View[0];
+        this._value = Arinc429Word.f32View[1];
+    }
+
+    static empty() {
+        return new Arinc429Word(0);
+    }
+
+    static fromSimVarValue(name) {
```

Comment:
Are we sure that creating a new instance of this class every update is a good idea ? I don't know if this design is final or not, considering this is a draft, but it seems to me that creating a bunch of objects we immediately stop using and re-create might put a lot of pressure on GC.

Perhaps an instance should be able to be "recomputed" when new data is available, without instantiating this class again.

Reply:
For now I'm sure this is a good idea, because it leads to the simplest programming interface which nearly directly replaces `SimVar.GetSimVarValue()`. Of course this isn't yet programmed for React, and there we might be able to do something smarter in a hook combined with `useSimVar`.